Apromore-Core-Components/Apromore-Manager/src/main/resources/META-INF/spring/managerContext-jpa.xml

```diff
@@ -106,5 +106,13 @@
           class="org.springframework.cache.ehcache.EhCacheManagerFactoryBean">
         <property name="configLocation" value="${manager.ehcache.config.url}"/>
     </bean>
+    
+     <bean id="liquibase"
```

Comment:
Please remove the extra space at the beginning of the opening bean tag

Reply:
lol there is only 1 space, Not sure why its showing 2 in here